src/lib/core/TLVReader.cpp

```diff
@@ -49,6 +49,12 @@ using namespace chip::Encoding;
 
 static const uint8_t sTagSizes[] = { 0, 1, 2, 4, 2, 4, 6, 8 };
 
+TLVReader::TLVReader() :
+    ImplicitProfileId(kProfileIdNotSpecified), AppData(nullptr), mElemLenOrVal(0), mBackingStore(nullptr), mReadPoint(nullptr),
```

Comment:
Should these be using member initializers where the members are declared instead?

Reply:
No, this is what causes a large size flash increase due to the inlining of the stack initialization directly, rather than calling a constructor.

Reply:
Note that you still use member initializers and just define the c-tor in cpp as:
```
TLVReader::TLVReader() = default;
```
And then it won't be inlined.